cdap-ui/app/cdap/components/AbstractWidget/MultipleValuesWidget/index.tsx

```diff
@@ -74,3 +76,11 @@ export default function MultipleValuesWidget(props) {
 }
 
 (MultipleValuesWidget as any).propTypes = WIDGET_PROPTYPES;
+(MultipleValuesWidget as any).getWidgetAttributes = () => {
+  return {
+    placeholders: { type: 'string[]', required: false },
+    'values-delimiter': { type: 'string', required: false },
+    numValues: { type: 'number', required: true },
```

Comment:
it looks like this can also be a string?

Reply:
Because this widget property `numValues` represents a number of values, we should render a number widget that configures this widget property. Therefore, it would be better to pass in `number` instead.